test/views/category_view_test.exs

```diff
@@ -4,37 +4,33 @@ defmodule CodeCorps.CategoryViewTest do
   alias CodeCorps.Repo
 
   # Bring render/3 and render_to_string/3 for testing custom views
-  import Phoenix.View
+  import Phoenix.View, only: [render: 3]
 
   test "renders all attributes and relationships properly" do
-    project_category = insert(:project_category)
-
-    category =
-      CodeCorps.Category
-      |> Repo.get(project_category.category_id)
-      |> CodeCorps.Repo.preload([:project_categories])
+    category = insert(:category)
+    project_category = insert(:project_category, category: category)
 
     rendered_json =  render(CodeCorps.CategoryView, "show.json-api", data: category)
 
     expected_json = %{
-      data: %{
-        attributes: %{
+      "data" => %{
+        "attributes" => %{
           "description" => category.description,
           "name" => category.name,
           "slug" => category.slug
         },
-        id: category.id |> Integer.to_string,
-        relationships: %{
+        "id" => category.id |> Integer.to_string,
+        "relationships" => %{
           "project-categories" => %{
-            data: [
-              %{id: project_category.id |> Integer.to_string, type: "project-category"}
+            "data" => [
+              %{"id" => project_category.id |> Integer.to_string, "type" => "project-category"}
             ]
           }
         },
-        type: "category",
+        "type" => "category",
       },
-      jsonapi: %{
-        version: "1.0"
+      "jsonapi" => %{
+        "version" => "1.0"
       }
     }
```

Comment:
A change in `ja_serializer`. They just do it this way, always, now. All keys are strings. I guess because it's closer to actual JSON.